packages/client/src/__tests__/integration/errors/multi-schema/test.ts

```diff
@@ -14,9 +14,9 @@ const title = faker.name.jobTitle()
 const newEmail = faker.internet.email()
 const newTitle = faker.name.jobTitle()
 
-describe('multi-schema', () => {
```

Comment:
Small rename along the way to make everything consistent

Reply:
Yeah, probably update it to `multischema/test.ts`

Reply:
indeed, I'll do this after